src/test/run-make/coverage/coverage_tools.mk

```diff
@@ -0,0 +1,6 @@
+# Common Makefile include for Rust `run-make/coverage-* tests. Include this
+# file with the line:
+#
+# -include ../coverage/coverage_tools.mk
+
+-include ../../run-make-fulldeps/tools.mk
```

Comment:
@RalfJung  @richkadel Is this a problem? Wasn't run-make-fulldeps renamed run-make?

using -include instead of include causes the error that it is missing to be ignored

I'm not sure how this would relate to the build failure though unless it is a Sinclude issue with a non-gnu make.

Reply:
@iladin The file is still there: https://github.com/rust-lang/rust/blob/3fca198e468e5c311eb97b70d6ce3af1dddc56e8/src/test/run-make-fulldeps/tools.mk

Reply:
Sorry, I have no idea how the run-make tests work -- the only reason I am involved in this discussion is that I was doing rollups and trying to figure out which PR broke the build. ;)

The other candidate just passed bors, so I am reasonably certain now it is this one -- but since the failure comes fast, there's no harm in letting bors try this PR to confirm.

Reply:
Just FYI, there are 25 other `run-make` tests that include the same file, with the same `-include` directive. I don't think that line is an issue.